tests/_benderjs/ckeditor/static/extensions.js

```diff
@@ -185,6 +185,35 @@
 		}
 	};
 
+	/**
+	 * Assertion function which uses {@link bender.editorBot.combo}, apply style provided by this combo and compare result.
+	 *
+	 * @param {Object} options configuration object
+	 * @param {String} options.comboName name of {@link CKEDITOR.ui.richCombo} added by {@link CKEDITOR.ui.addRichCombo} to editor.
+	 * @param {String} options.comboValue value selected from rich combo
+	 * @param {Boolean} options.collapsed information if selection is collapsed. If it's true, then additional text is inserted to not remove empty span from some browsers.
+	 * @param {Object} options.bot bot instance in current test case
+	 * @param {String} options.resultHtml html which should be present after applying rich combo change
+	 */
+	bender.assert.assertCombo = function( options ) {
```

Comment:
Actually I was thinking about moving it to the richcombo testing tools, because it is generic, but not _that_ generic ;) Comparing to the other Bender extensions, it's clear that this one is connected with certain plugin.